website/routes.py

```diff
@@ -1150,6 +1156,11 @@ def make_url_map(app):
         ], 'get', project_views.register.node_register_template_page, json_renderer),
 
         Rule([
+            '/project/<pid>/retract_registration/',
+            '/project/<pid>/node/<nid>/retract_registration/'
+        ], 'post', project_views.register.node_registration_retraction, json_renderer),
```

Comment:
Disregard. If we go with ` /project/<pid>/retraction/` as mentioned above, then `POST` makes more sense.